services/ingestion/etl/agri/csv/test/unit/events/onParseCSV.spec.js

```diff
@@ -1,15 +1,16 @@
-import { promisify } from 'util';
 import onParseCSV from '../../../src/events/onParseCSV';
 
-const handler = promisify(onParseCSV);
-
 describe(`Function parseCsv in "@eubfr/ingestion-etl-agri-csv"`, () => {
-  test('The function expects a correct SNS record', () => {
+  test('The function expects a correct SNS record', async () => {
```

Comment:
I don't mind having async/await, but let's discuss converting the codebase somehow?
It's getting very colourful the codebase with all sorts of async patterns :)

Reply:
Agree.
I've started playing with async/await patterns here and it feels much cleaner than promises, it's often easier to read. I would suggest that we start using this pattern more frequently, when it makes more sense than using promises